hilti/toolchain/include/ast/declarations/parameter.h

```diff
@@ -133,4 +133,10 @@ inline bool areEquivalent(Parameter* p1, Parameter* p2) {
     return type::same(p1->type(), p2->type());
 }
 
+/** Returns true if two sets of parameters are equivalent, regardless of their ID. */
+inline bool areEquivalent(node::Set<Parameter> params1, node::Set<Parameter> params2) {
+    return std::equal(std::begin(params1), std::end(params1), std::begin(params2), std::end(params2),
```

Comment:
Nit: We don't need non-member `begin`/`end` here and using it makes the line longer.

```suggestion
    return std::equal(params1.begin(), params1.end(), params2.begin(), params2.end(),
```